### ISSUE TITLE
[Usability][Screen reader-Welcome] Nvda is reading twice for check boxes information 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1823]
**Actual Result:**
Nvda is reading twice for checkboxes information present in the welcome tab screen
NVDA is reading as “Open in Debug mode Open in Debug mode check box not checked”
1.Open in Debug mode 2. Azure for US Government  

**Expected Result:**
 NVDA should narrate the once for checkboxes information 
Example: NVDA should read as “Open in Debug mode check box checked \Not checked “

**Note:** This issue exits throughout the application for all check boxes 
Same issue repro for Edit ngrok settings  screen 
"User settings,Application updates,Data collection" under check boxes  NVDA is reading the twice 

Same issue repro for below path 
Welcome Bot Framework Emulator ->Create new bot configuration ->Live chat dialog 
Type your message text is reading the twice 
Please refer the attachment:Usability_Live.chat.zip

**Test Environment:**
OS Build: Version 1903(OS Build 18362.239)
Application: Bot Framework V4 Emulator
Screen reader: NVDA

**Repro Steps:**
Turn on the NVDA 
1.	Open the Bot Framework V4 Emulator.
2.	Navigate to the Open bot control and select it 
3.	Dialog gets opened 
4.	 Navigate to the “Open in Debug mode and Azure for US Government ” check boxes 
5.	Observe the issue 

**User Impact:**
Users who rely on screen reader will be affected to navigate the check boxes if screen reader reading the same info twice then the screen reader users will misguide information.
[Usaibility_Open a bot.zip](https://github.com/microsoft/BotFramework-Emulator/files/3574309/Usaibility_Open.a.bot.zip)

[Usability_Live chat.zip](https://github.com/microsoft/BotFramework-Emulator/files/3578531/Usability_Live.chat.zip)

**Comment by Kumar2608** [comment]
Verified in latest Bot emulator version -4.5.2-79718 with NVDA(2019.2) issue partially fixed. hence we are reactivating this bug.
For note section issue  still repro'ing .
Please refer the above attachment: Usability_Live Chat. Zip
Welcome Bot Framework Emulator ->Create new bot configuration ->Live chat dialog
Type your message text is reading the twice

**Comment by corinagum** [comment]
I will be implementing the fix for the sendbox ("Type your message") being read twice on Web Chat. Therefore my fix will not be available to Emulator until Web Chat is bumped, which should be at the same time that #1759 is fixed. For reference I will link the Web Chat PR that contains this fix below.

**Comment by corinagum** [comment]
This was closed on accident when my PR was merged in Web Chat, above. Keeping this open intentionally until Web Chat is bumped on Emulator.

**Comment by cwhitten** [comment]
This is addressed in the WebChat dependency via the following PR: https://github.com/microsoft/BotFramework-WebChat/pull/2423. We will take a new version of WebChat in the coming weeks.
